src/installCommand/index.js

```diff
@@ -132,6 +145,33 @@ async function lookupPackageCollection(packageName: string): Promise<PackageJson
   return await readJson(packageRecordFilename);
 }
 
+async function fetchFromOpam(target, resolution) {
+  if (resolution.tarball === 'empty') {
+    await mkdirp(target);
+  } else {
+    const filename = path.basename(resolution.tarball);
+    const stage  = tempfile(filename);
+    await mkdirp(stage);
+    await mkdirp(target);
+    await execa('wget', [resolution.tarball], {cwd: stage});
```

Comment:
Will everyone have `wget`? I have it installed via brew for some reason, so that makes me thing it's not installed by default on OS X or some variants of Linux.